NuRadioReco/framework/trigger.py

```diff
@@ -103,7 +103,19 @@ def set_trigger_time(self, time):
 
     def get_trigger_time(self):
         """
-        get the trigger time (absolute time with respect to the beginning of the event)
+        Get the trigger time
+
+        Returns the absolute time of the trigger with respect to the beginning of the event,
+        i.e. the first time in the event where the trigger condition was fulfilled
```

Comment:
The _absolute time_ of the trigger _with respect to_ the beginning of the event sounds like a contradiction by itself to me. If it relative to e.g., the `station_time` or `event_time` I would specify this.